tools/bouncer_worker/src/tasks/bouncerClient.js

```diff
@@ -30,6 +30,10 @@ const setBouncerEnvars = (logDir) => {
 		});
 	}
 
+	if (config.bouncer.envpath) {
```

Comment:
do you really need this?

for me and devops paths are already set to run elsewhere. This seems a bit specific to your build?

Reply:
No I don't think its necessary anymore - removed.